code/modules/jobs/job_types/assistant.dm

```diff
@@ -21,6 +21,13 @@ Assistant
 
 	family_heirlooms = list(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/gloves/cut/heirloom)
 
+	mail_goodies = list(
+		/obj/effect/spawner/lootdrop/donkpockets = 10,
+		/obj/item/clothing/mask/gas = 10,
+		/obj/item/clothing/gloves/color/yellow = 5,
```

Comment:
I think a small chance for actual insulated gloves could be fun to balance out the more common fake gloves

Reply:
aren't those the real ones though

Reply:
Those are the real deal, yes. Going to do a tweak on these percentages, though.